packages/sirius-web/frontend/sirius-web-application/src/views/edit-project/EditProjectView.tsx

```diff
@@ -126,7 +126,9 @@ export const EditProjectView = () => {
     content = (
       <ProjectContext.Provider value={{ project: context.project }}>
         <SelectionContextProvider initialSelection={initialSelection}>
-          <OmniboxProvider initialContextEntries={initialContextEntries}>
+          <OmniboxProvider
+            initialContextEntries={initialContextEntries}
+            editingContextId={context.project.currentEditingContext.id}>
```

Comment:
You are duplicating a value that you have one line above (but the current API sucks)

Reply:
True, the search command won't be there if there is no `EditingContext` context entry anyways. It just felt odd to me to have the search query rely on the existence of the correct context entry, and that all this is tied up by a condition in the command provider in the backend. But a better API can probably be done later.